Dockerfile

```diff
@@ -0,0 +1,29 @@
+# gammapy docker container
+FROM continuumio/miniconda3:latest
+RUN apt-get update && apt-get install -y build-essential python3-pip
+
+# define workdir
+WORKDIR /usr/src/app/
+
+# copy content
+COPY . /usr/src/app/
+
+# create conda env
+RUN conda install -c conda-forge mamba
+RUN mamba env create -n gammapy -f environment-dev.yml
+RUN echo 'source activate gammapy' >> ~/.bashrc
+ENV PATH /opt/conda/envs/gammapy/bin:$PATH
+
+# install gammapy
+RUN pip install -e .
```

Comment:
If this is for reproducibility, wouldn't it make more sense to just install a specific version of gammapy and specific versions of all needed packages instead of doing a development install?